doc/developers/contributing.rst

```diff
@@ -269,7 +269,7 @@ how to set up your git repository:
 
    .. prompt:: bash
 
-        pip install pytest pytest-cov ruff mypy numpydoc black==24.3.0
+        pip install pytest pytest-cov ruff mypy numpydoc
```

Comment:
we should mention the ruff version here then, like black

Reply:
We did not mention the version of black the linter. Do we want to mention the version of ruff the formatter? It seems to me that the formatter is stable enough these days to not worry about pinning ruff.

Reply:
In any case, I would recommend merging #30976 first, to update ruff. It's better to start with the more stable formatter 0.11.0 instead of the obsolete 0.5.1.

Reply:
Pinned to latest ruff 0.11.2.